cypress/integration/rendering/journey.spec.js

```diff
@@ -64,3 +64,100 @@ section Checkout from website
     );
   });
 });
+
+describe('User journey diagram task score behavior validation', () => {
+  it('should throw an error if the task score is non-integer', () => {
+    let errorCaught = false;
+
+    cy.once('uncaught:exception', () => {
+      errorCaught = true;
+      return false;
+    });
+
+    renderGraph(`
+      journey
+      accTitle: simple journey demo
+      accDescr: 2 main sections: work and home, each with just a few tasks
+
+      section Go to work
+        Make tea: Hello: Me
+        Go upstairs: 3: Me
+      section Go home
+        Go downstairs: 5: Me
+        Sit down: 2: Me
+    `);
+
+    cy.wait(500).then(() => {
+      expect(errorCaught, 'Error should be thrown for a non-integer score').to.equal(true);
+    });
```

Comment:
It's not ideal to have time based waits in tests, can you use an event based wait?